### DISCUSSION TITLE
Focus on error on custom select with hidden input?

### DISCUSSION THREAD

**Discussion by christian-reichart** [airjp73/rvf#326] [Q&A] [answered]
Hi guys, I have the following issue:

I have a special built select field that works with a span and a hidden input. Now when this field has an error the auto focus doesn't work. Do you have any idea how I could get this to work?

I'm working with [Reach Listbox](https://reach.tech/listbox/)

**Comment by airjp73** [comment, marked as answer]
`useField` has a `handleReceiveFocus` option specifically for this type of case.

https://www.remix-validated-form.io/reference/use-field#handleReceiveFocus

**Reply by christian-reichart** [reply]
@airjp73  oh wow I missed this, thanks so much for the fast reply!
